ote/src/cljs/ote/app/controller/transit_visualization.cljs

```diff
@@ -81,15 +81,26 @@
                      detection-date))))
       changes)))
 
+(defn count-changes [key coll]
+  (count
+    (get coll key)))
 
 (defn sorted-route-changes
   "Sort route changes according to change date and route-long-name: Earliest first and missing date last."
   [show-no-change changes]
   (let [;; Removed in past routes won't be displayed at the moment. They are ended routes and we do not need to list them.
         ;removed-in-past (sort-by (juxt :route-long-name :route-short-name) (filterv #(and (= :removed (:change-type %)) (nil? (:change-date %))) changes))
         no-changes (sort-by (juxt :route-long-name :route-short-name) (filterv #(= :no-change (:change-type %)) changes))
-        only-changes (filterv :change-date changes)
-        sorted-changes (sort-by (juxt :different-week-date :route-long-name :route-short-name) only-changes)
+        only-changes (sort-by :different-week-date (filterv :change-date changes))
```

Comment:
Should this include also :no-traffic* changes and not ony :different-week* changes?

Reply:
As discussed irl. different-week-date is correct key.